api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -1736,6 +1736,14 @@ type AzurePlatformSpec struct {
 	// +immutable
 	ResourceGroupName string `json:"resourceGroup"`
 
+	// ClientResourceGroupName is the name of an existing resource group provided by a client; it is expected to contain
+	// an existing, pre-setup VNET and Subnet. This field is only used by ARO HCP.
```

Comment:
> This field is only used by ARO HCP.

The field is in the `AzurePlatformSpec` - do we need the callout for ARO?

Reply:
Yeah, ARO HCP prefers to know which fields would be applicable to ARO HCP vs self managed Azure.